ambrs/aerosol.py

```diff
@@ -89,6 +90,7 @@ class AerosolModeState:
     def mass_fraction(self, species_name: str):
         """returns the mass fraction corresponding to the given species name,
 or throws a ValueError."""
+        print(self.species)
```

Comment:
We can probably get rid of this `print`.

Reply:
Ah yes. I was debugging. Thanks!